src/current/v23.1/change-data-capture-overview.md

```diff
@@ -6,49 +6,85 @@ docs_area: stream_data
 
-Refer to [Ordering Guarantees](changefeed-messages.html#ordering-guarantees) for detail on CockroachDB's at-least-once-delivery-guarantee as well as explanation on how rows are emitted.
+## Get started with changefeeds
+
+To get started with changefeeds in CockroachDB, refer to:
+
+- [Create and Configure Changefeeds](create-and-configure-changefeeds.html): Learn about the fundamentals of using SQL statements to create and manage Enterprise and Core changefeeds.
+- [Changefeed Sinks](changefeed-sinks.html): The downstream system to which the changefeed emits changed row data. Learn about the supported sinks that Enterprise changefeeds can emit to.
+- [Changefeed Messages](changefeed-messages.html): The change events that emit from the changefeed to your sink. Learn about how messages are ordered at your sink and the options to configure and format messages.
+- [Changefeed Examples](changefeed-examples.html): Step-by step-examples for connecting to each changefeed sink.
+
+### Authenticate to your changefeed sink
+
+To send changefeed messages to a sink, it is necessary to provide the `CREATE CHANGEFEED` statement with authentication credentials. The following pages detail the supported authentication:
+
+Sink | Authentication page
+-----+---------------------
+Cloud Storage | Refer to [Cloud Storage Authentication](cloud-storage-authentication.html) for detail on setting up:<br>IAM roles with assume role authentication.<br>Workload identity authentication.<br>Implicit authentication.<br>Specified authentication.
+Kafka | Refer to:<br>[Connect to a Changefeed Kafka Sink with OAuth Using Okta](connect-to-a-changefeed-kafka-sink-with-oauth-using-okta.html) to connect to your Kafka sink using OAuth authentication.<br>[Stream a Changefeed to a Confluent Cloud Kafka Cluster](stream-a-changefeed-to-a-confluent-cloud-kafka-cluster.html) to authenticate to a Confluent Cloud Kafka cluster with a Confluent Schema Registry.<br>[Query parameters](changefeed-sinks.html#kafka) to supply your own certificate credentials.
+Webhook | [Query parameters](changefeed-sinks.html#webhook-sink) to supply your own certificate credentials.
+Google Cloud Pub/Sub | Refer to:<br>[Query parameters](changefeed-sinks.html#google-cloud-pub-sub) for a general list of the supported query parameters.<br>[Cloud Storage Authentication](cloud-storage-authentication.html?filters=gcs#google-cloud-storage-assume-role) for detail on IAM roles with assume role authentication.
+
+## Optimize a changefeed for your workload
+
+### Monitor your changefeed job
```

Comment:
Monitoring feels more like it belongs in a section by itself rather than as part of the "optimize" section. Love that as a section for the rest, btw!